SysTrayIcon/explainer.md

```diff
@@ -0,0 +1,149 @@
+# **System Status Icons for Installed desktop Web Apps**
+
+Authors: [Diego Gonzalez](https://github.com/diekus), [Jenna Sasson](https://github.com/jennasasson_microsoft)
+
+## Status of this Document
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+* This document status: **Active**
+* Expected venue: [W3C Web Applications Working Group](https://www.w3.org/groups/wg/webapps/)
+* **Current version: this document**
+
+##  Introduction
+Desktop platform-specific applications have several ways of indicating that they are running. On platforms like Windows and macOS, the corresponding app icon on the taskbar/dock displays a little dot indicating that the app is open (or has an open window) and running. Additionally, *open applications may also have an icon on a specific part of the OS where users glance at notifications or system status (settings/shortcuts)*. This is very useful for end users because these icons can be used as a re-entry point to the app and can also convey additional information and options to them.
+
+This specific area varies in names depending on the platform, but the functionality for this icon across platforms is the same. Taken from the Apple's Human Interface Guidelines, this feature "exposes app-specific functionality using an icon that appears in the menu bar when your app is running, even when it’s not the frontmost app". This is called a menu extra in macOS, and sits next to the OS's *status menus*. On Windows, it is called a system tray icon, and sits in the notification area (aka system tray). We will call this area the ***system status area***. 
```

Comment:
Can you add a link to the part of the Human Interface Guidelines and ideally the Windows equivalent?

Reply:
the links are there, I think I've posted them further down though :p

I've added them in that particular section (intro), where you can access the concept of notification icon and menu bar extra.